flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -292,7 +360,76 @@ public void registerKryoType(Class<?> type) {
 	/**
 	 * Returns the registered POJO types.
 	 */
-	public Set<Class<?>> getRegisteredPojoTypes() {
+	public List<Class<?>> getRegisteredPojoTypes() {
 		return registeredPojoTypes;
 	}
+
+
+	public boolean isDisableAutoTypeRegistration() {
+		return disableAutoTypeRegistration;
+	}
+
+	/**
+	 * Control whether Flink is automatically registering all types in the user programs with
+	 * Kryo.
+	 *
+	 * @param disableAutoTypeRegistration
+	 */
+	public void setDisableAutoTypeRegistration(boolean disableAutoTypeRegistration) {
```

Comment:
I would prefer "disableAutoTypeRegistration" here. And then "isAutoTypeRegistrationDisabled", above.